### ISSUE TITLE
Does not work with flatpak version of Discord

### ISSUE THREAD

**Issue by rambo919** [darktohka/audacious-plugin-rpc#14]
Given how new that version is it's not surprising but given how bad regular discord is at updating itself I expect everyone will be switching to the flatpak version.

**Comment by onegen-dev** [comment]
I don’t think that is a concern of this project, as it uses the standard RPC library. If Flatpak Discord is not detecting RPCs, some extra steps are needed to make it work – there’s a good [ArchWiki page](https://wiki.archlinux.org/title/Discord#Enabling_rich_presence_on_Flatpak) about that.

> *how bad regular discord is at updating itself*

Painfully know the feeling, on Linux the forced updates are hella annoying. One day I just discovered I can [stop the updates altogether](https://superuser.com/a/1805677), so… there is that. :P
